Lib/asyncio/runners.py

```diff
@@ -45,10 +45,11 @@ class Runner:
 
     # Note: the class is final, it is not intended for inheritance.
 
-    def __init__(self, *, debug=None, loop_factory=None):
+    def __init__(self, *, debug=None, loop_factory=None, task_factory=None):
```

Comment:
Do we really need a separate `task_factory` here?  Ultimately this is just influencing the loop after it's created, couldn't that be rolled into the existing `loop_factory` mechanism?

Reply:
no we don't need this. it was convenient to add it for my testing. sure, the same result can be achieved with the loop factory, but this feels cleaner, so I figured I'd suggest it and see what others think